### PR TITLE
Editor ingress settings

### PR DESCRIPTION
**Pull request by elenaviter**
## Description

Complementary PR for ingress settings and Projects instances tolerations settings configured in FlowForge helm to be propagated to Editor instances created by FlowForge. 

## Related Issue(s)

 - [Ingress settings for Flowforge in K8s - Helm](https://github.com/flowforge/helm/pull/125)
 - [Ingress settings for k8s driver](https://github.com/flowforge/flowforge-driver-k8s/pull/80)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated
    - [ ] Upgrade instructions
    - [x] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by hardillb** [comment]
@andreikop I assume you have co-ordinated with @elenaviter on this and as such #125 is obsolete?

I will look at this (and the k8s driver PR) in more detail later, but I should note that we have moved away from using AWS ALB ingress for our deployment, because AWS ALB has a hard limit of only supporting 100 entry points (so 98 projects + forge app and MQTT broker) so does not scale. We have moved to using nginx ingress and just setting default ingressClass to get everything working.

The docs need updating and this is in plan for the 1.6 release.

**Comment by andreikop** [comment]
@hardillb yes, but we make atomic PRs to make them easier to understand and apply.

**Comment by hardillb** [comment]
Just a quick note, we have not forgotten about these 2 PRs, but having looked at the kubernetes driver PR (https://github.com/flowforge/flowforge-driver-k8s/pull/80) that they depend on last night and finding it needs some changes we are going to miss the 1.6.0 release today. Assuming everything is cleaned up, we will look to get them merged early in the 1.7.0 cycle.

**Comment by hardillb** [comment]
@elenaviter Can you rebase this on head to pick up the changes I asked for in @andreikop's PR then I can look at merging this and the changes to the k8s driver.

**Comment by hardillb** [comment]
@elenaviter Any chance of getting this rebased today please so we can get it merged and tested ahead of the release next week?

File: helm/flowforge/values.schema.json [hunk 1]
```diff
@@ -248,8 +248,8 @@
                             "type",
                             "options"
                         ]
-                    } 
-                    
+                    }
+
```
Comment (hardillb) [inline helm/flowforge/values.schema.json:252]:
```suggestion
                    }
```

File: helm/flowforge/values.yaml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 forge:
+  domain: "TO FILL"
+  entryPoint: "TO FILL"
+  registry: "TO FILL"
```
[the remark this answers is not in the document]
Comment (hardillb) [reply helm/flowforge/values.yaml:4]:
```suggestion
```

File: helm/flowforge/values.schema.json [hunk 3]
```diff
@@ -248,8 +248,8 @@
...
                 },
```
Comment (hardillb) [inline helm/flowforge/values.schema.json:253]:
```suggestion
                    }
                },
```

**Review by hardillb — APPROVED** [review]

**Pull request merged**